grazer/src/main/resources/locale/messages_en_US.properties

```diff
@@ -1 +1,4 @@
-welome_msg=Hello in English!
\ No newline at end of file
+welome_msg=Hello in English!
+NotEmpty.transformation.label = Label is required!
+NotEmpty.transformation.transformationFile = Upload transformation file!
```

Comment:
Let's be a little less bossy here and say "A transformation file is required." and "A pattern file is required." ;)